scripts/watch.js

```diff
@@ -100,6 +101,12 @@ const setupPreloadPackageWatcher = ({ws}) =>
     name: 'reload-page-on-preload-package-change',
     configFile: 'packages/preload/vite.config.js',
     writeBundle() {
+      // Generating exposedInMainWorld.d.ts when preload package is changed.
+      generate({
```

Comment:
Maybe you should use `generateAsync` here

Reply:
Good idea！🍺